perl6.html.markdown

```diff
@@ -193,6 +198,15 @@ sub mutate($n is rw) {
   say "\$n is now $n !";
 }
 
+my $m = 42;
+mutate $m; # $n is now 43 !
+
+# This works because we are passing the container $m to mutate.  If we try
+# to just pass a number instead of passing a variable it won't work because
+# there is no container being passed and integers are immutable by themselves:
+
+mutate 42; # Parameter '$n' expected a writable container, but got Int value
```

Comment:
I posted above: Also, what best way to resync my repo with yours so that the old commits that have already been merged don't continue to show up? Thanks!